crates/bevy_gltf/src/loader.rs

```diff
@@ -479,14 +493,19 @@ async fn load_gltf<'a, 'b, 'c>(
                 && primitive.material().normal_texture().is_some()
             {
                 bevy_log::debug!(
-                    "Missing vertex tangents, computing them using the mikktspace algorithm"
+                    "Missing vertex tangents for {}, computing them using the mikktspace algorithm. Consider using a tool such as Blender to pre-compute the tangents.", file_name
                 );
-                if let Err(err) = mesh.generate_tangents() {
-                    warn!(
+
+                let generate_tangents_span = trace_span!("generate_tangents", name = file_name);
```

Comment:
As far as I can tell every other span in Bevy is `info_span!` so I suggest using that.